scripts/destroy.sh

```diff
@@ -1,21 +1,28 @@
 #!/bin/bash
 NAMESPACES="flux control delivery logging monitoring security tracing"
 
-# Remove Helm Operator
-helm delete --purge helm-operator
+kubectl delete --wait=true -R -f platform/flux/
 
-# Remove Tiller
-helm reset --force
+# Delete all helm Releases and corresponding resources
+# along with release history in kube-system
+helm delete --purge $(helm list --short)
+
+# Remove any remaning platform resources
+kubectl delete --wait=true --ignore-not-found -R -f platform/
 
-# Delete all HRs
+# Remove remaining PVCs in all namespaces
 for NAMESPACE in $NAMESPACES; do
-  kubectl delete hr --all -n $NAMESPACE
+  kubectl delete pvc --all -n $NAMESPACE
 done
 
-# Remove any remaing platform resources
-kubectl delete -f platform/
+# Remove Helm Operator
+helm delete --purge helm-operator
+
+# Remove Tiller
+helm reset --force
 
-#TODO remove role and cluster roles
+# Manually delete helmRelease CRD. Temp fix. Will delete all crds.
```

Comment:
@usamaahmadkhan  none of the release is deleted automatically or few?

Reply:
None of the crds are deleted